synapse/federation/federation_server.py

```diff
@@ -515,7 +515,7 @@ async def process_pdu(pdu: EventBase) -> JsonDict:
                     logger.error(
                         "Failed to handle PDU %s",
                         event_id,
-                        exc_info=(f.type, f.value, f.getTracebackObject()),  # type: ignore
+                        exc_info=(f.type, f.value, f.getTracebackObject()),
```

Comment:
I don't really see how this is relevant to warn_unused_configs: https://mypy.readthedocs.io/en/stable/config_file.html#confval-warn_unused_configs makes it sound like the option tells you when you've screwed up your config.

(Happy to remove any redundant type-ignores though.)

Reply:
I also don't really understand how it is relevant, but when I enabled that option it gave errors on these lines? 🤷